src/pal/src/loader/module.cpp

```diff
@@ -1628,6 +1636,12 @@ static HMODULE LOADLoadLibrary(LPCSTR shortAsciiName, BOOL fDynamic)
     LockModuleList();
 
     dl_handle = LOADLoadLibraryDirect(shortAsciiName);
+#ifdef __APPLE__
+    if (!dl_handle && isLibcRequested)
+    {
+        dl_handle = LOADLoadLibraryDirect("/usr/lib/libc.dylib");
```

Comment:
Is this really needed as fallback path? Would it be better to change the libc path above to `/usr/lib/libc.dylib` for Apple?